ci-operator/step-registry/acm/tests/clc-create/acm-tests-clc-create-ref.yaml

```diff
@@ -0,0 +1,64 @@
+ref:
+  as: acm-tests-clc-create
+  from: clc-ui-e2e
+  commands: acm-tests-clc-create-commands.sh
+  timeout: 28800s
+  resources:
+    requests:
+      cpu: '1'
+      memory: 3Gi
+  credentials:
+    - namespace: test-credentials
+      name: acm-clc-credentials
+      mount_path: /tmp/secrets/clc
+  env:
+  - name: CYPRESS_OC_IDP
+    default: "kube:admin"
+    documentation: |-
+      Identity
+  - name: CYPRESS_OPTIONS_HUB_USER
+    default: "kubeadmin"
+    documentation: |-
+      Hub cluster username
+  - name: CYPRESS_SPOKE_CLUSTER
+    default: ""
+    documentation: |-
+      Identify spoke clusters
+  - name: BROWSER
+    default: "chrome"
+    documentation: |-
+      Set browser for cypress
+  - name: CUSTOMER_TAGS
+    default: ""
+    documentation: |-
+      Freestyle tags for cypress
+  - name: CYPRESS_CLC_OC_IDP
+    default: "clc-e2e-htpasswd"
+    documentation: |-
+      Serves tests RBAC settings
+  - name: CYPRESS_CLC_RBAC_PASS
+    default: "test-RBAC-4-e2e"
+    documentation: |-
+      Serves tests RBAC settings
+  - name: CYPRESS_CLC_OCP_IMAGE_VERSION
```

Comment:
Isn't this just overridden in the script?

Reply:
Yes it is I can remove this from the env section, good catch!